cli/src/commands/squash.rs

```diff
@@ -41,7 +41,7 @@ use crate::description_util::join_message_paragraphs;
 use crate::description_util::try_combine_messages;
 use crate::ui::Ui;
 
-/// Move changes from a revision into another revision
+/// Move changes from a revision into another revision [default alias: amend]
```

Comment:
I'm less sure about this one since the alias exists just to ease the transition from `hg amend` (I don't think there's a `git amend`). Thoughts?

Reply:
It's gone now.